src/Redux/matchSlice.js

```diff
@@ -0,0 +1,32 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const serverURL = process.env.REACT_APP_SERVER;
+
+const initialState = { entities: {}, loading: 'idle' };
+
+export const fetchMatches = createAsyncThunk(
+  'match/fetchMatches',
+  async (data) => {
+    try {
+      const response = await axios({
+        method: 'POST',
+        headers: { 'x-auth-token': localStorage.getItem('x-auth-token') },
+        url: `${serverURL}/matches/oneFilm`,
```

Comment:
it would be cool to set these headers globally, I'm not really sure how to do that but I'd be cool with changing over to do it that way.  As for the global baseUrl I tried just using the 'matches/oneFilm' format and to my understanding this would mean the call would default to the base url of the website but I got kinda inconsistent results and so I ended up doing it this way since it was reliable.  I'm not sure what I was doing wrong but I'd be fine with changing it, as long as it works XD!